src/ng/q.js

```diff
@@ -381,7 +381,11 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
       if (!toCheck.pur) {
         toCheck.pur = true;
         var errorMessage = 'Possibly unhandled rejection: ' + toDebugString(toCheck.value);
-        exceptionHandler(errorMessage);
+        if (toCheck.value instanceof Error) {
+          exceptionHandler(toCheck.value, errorMessage);
```

Comment:
OK, fair enough. In that case, maybe it's better to leave it as it is for now; your proposal to just write:
```js
exceptionHandler(toCheck.value, 'Possibly unhandled rejection');
```
sounds nice but we pass other data types through `toDebugString` and we'd lose it in that case. If we want to tweak that, it would be better to tackle that separately after this PR lands.